packages/oruga/src/components/autocomplete/tests/SimpleAutocomplete.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+import { ref } from "vue";
+
+import OAutocomplete from "../Autocomplete.vue";
+import OField from "../../field/Field.vue";
+
+const options = [
+    "Angular 2",
+    "Ember",
+    "jQuery",
+    "Meteor",
+    "Node.js",
+    "React",
+    "RxJS",
+    "Vue.js",
+];
+const name = ref("");
+const selected = ref<string>();
+</script>
+
+<template>
```

Comment:
Maybe you can convert this component into an inline component, like you can see in `carousel.axe.test.ts`. so we reduce the need for an extra file and have everthing fo the test in one file :)